charts/postgres-operator/values-crd.yaml

```diff
@@ -93,10 +93,9 @@ configKubernetes:
   # template for database user secrets generated by the operator
   secret_name_template: "{username}.{cluster}.credentials.{tprkind}.{tprgroup}"
   # group ID with write-access to volumes (required to run Spilo as non-root process)
-  # spilo_fsgroup: 103
-
+  spilo_fsgroup: "103"
   # whether the Spilo container should run in privileged mode
-  spilo_privileged: false
+  spilo_privileged: "false"
```

Comment:
IIRC, CRD-config does not like quotes for bools. Please remove them.

Reply:
Are you sure about this?

I tested the changes in `manifests/configmap.yaml` where it fails when the values are not quoted. I assume the rest of the YAML files are using the same parser/schema?

Reply:
For the ConfigMap it's indeed necessary to quote such flags. That's why I only commented on the CRD-baed config changes ;)